src/source_map_utils.ts

```diff
@@ -30,7 +30,8 @@ export function extractInlineSourceMap(source: string): string {
     previousResult = result;
     result = inlineSourceMapRegex.exec(source);
   } while (result !== null);
-  const base64EncodedMap = previousResult![1];
+  if (!previousResult) return null;
```

Comment:
I agree that it's bad to have an api that forces you to call a guard function before you call another function, but I don't think that what's happening here.  My goal with extractInlineSourceMap is that you only call it when you're sure that the file you're calling it on has an inline source map and that it throws nasty errors if you didn't figure that out correctly.

In the errors that this PR is a response to, there were in fact inline source maps present, but they weren't on their own line.  This PR just ignores them, instead of either fixing the source map regex or fixing whatever put the inline source map on a line with other stuff.

Is there some other way to get what I want (errors when there is in fact a source map, but something went wrong) as well as what you want (a simpler api)?